pkg/lib/indexer/indexer_test.go

```diff
@@ -0,0 +1,61 @@
+package indexer
+
+import (
+	"github.com/ghodss/yaml"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"sort"
+	"testing"
+
+	pregistry "github.com/operator-framework/operator-registry/pkg/registry"
+)
+
+func TestGetBundlesToExport(t *testing.T) {
+	expected := []string{"quay.io/olmtest/example-bundle:etcdoperator.v0.9.2", "quay.io/olmtest/example-bundle:etcdoperator.v0.9.0",
+		"quay.io/olmtest/example-bundle:etcdoperator.v0.6.1"}
+	sort.Strings(expected)
+
+	bundleImages, err := getBundlesToExport("./test/bundles.db", "etcd")
+	if err != nil {
+		t.Fatalf("exporting bundles from db: %s", err)
+	}
+	sort.Strings(bundleImages)
+
+	if !equal(expected, bundleImages) {
+		t.Fatalf("exporting images: expected matching bundlepaths: expected %s got %s", expected, bundleImages)
+	}
+}
+
+func equal(a, b []string) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i, v := range a {
+		if v != b[i] {
+			return false
+		}
+	}
+	return true
+}
+
+func TestGeneratePackageYaml(t *testing.T) {
+	err := generatePackageYaml("./test/bundles.db", "etcd", ".")
+	if err != nil {
+		t.Fatalf("writing package.yaml: %s", err)
+	}
+
+	var expected pregistry.PackageManifest
+	expectedResults, _  := ioutil.ReadFile("./test/package.yaml")
+	_ = yaml.Unmarshal(expectedResults, expected)
```

Comment:
I'm not sure it's safe to ignore errors from `Unmarshal`, even in a test. What if the decoder encounters an error after partially populating `expected`?